### PR TITLE
feat: Group Video Call

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: app/script/calling/entities/CallEntity.js [hunk 1]
```diff
@@ -257,30 +254,40 @@ z.calling.entities.CallEntity = class CallEntity {
 
   /**
    * Join the call.
-   * @returns {undefined} No return value
+   * @param {z.media.MediaType} [mediaType=z.media.MediaType.AUDIO] - Media type of the call
```
Comment (Yserz) [inline app/script/calling/entities/CallEntity.js:257]: What's the reason for `[mediaType=z.media.MediaType.AUDIO]`?

File: app/page/template/video-calling.htm [hunk 2]
```diff
@@ -1,110 +1,94 @@
...
-                 data-bind="click: clickedOnMinimize, css: {'button-round-dark': !showRemoteParticipant()}"></div>
-            <div class="video-controls-button button-round button-round-md icon-mute"
-                 data-uie-name="do-call-controls-video-call-mute"
-                 data-bind="click: clickedOnMuteAudio, css: {'toggled': !selfStreamState.audioSend(), 'button-round-dark': !showRemoteParticipant()}"></div>
-            <!-- ko if: showToggleScreen() -->
-              <div class="video-controls-button button-round button-round-md icon-screensharing"
-                   data-bind="click: clickedOnShareScreen, css: {'toggled': selfStreamState.screenSend(), 'button-round-dark': !showRemoteParticipant(), 'disabled': disableToggleScreen()}"
-                   data-uie-name="do-call-controls-toggle-screen"></div>
-            <!-- /ko -->
+          <div id="video-controls" class="video-controls" data-bind="css: {'hide-controls-hidden': !showRemoteParticipant()}">
```
Comment (atomrc) [inline app/page/template/video-calling.htm:33]: Note for later: Not sure this condition is still relevant `{'hide-controls-hidden': !showRemoteParticipant()}`.

File: app/script/calling/CallingRepository.js [hunk 3]
```diff
@@ -1031,6 +926,12 @@ z.calling.CallingRepository = class CallingRepository {
         amplify.publish(z.event.WebApp.WARNING.SHOW, z.viewModel.WarningsViewModel.TYPE.UNSUPPORTED_OUTGOING_CALL);
         throw new z.calling.CallError(z.calling.CallError.TYPE.NOT_SUPPORTED);
       }
+
+      const isVideoCall = mediaType === z.media.MediaType.AUDIO_VIDEO;
```
Comment (atomrc) [inline app/script/calling/CallingRepository.js:930]: Maybe we should check all the possible type that can be considered a video.

- AUDIO_VIDEO
- VIDEO
- SCREEN

File: app/script/calling/entities/CallEntity.js (hunk 1 above)
Comment (atomrc) [reply app/script/calling/entities/CallEntity.js:257]: Oh yeah, the default value has moved to the method `_joinGroupCall`.
https://github.com/wireapp/wire-webapp/pull/3450/files#diff-4816db425e94bac1930aae4a710ce13bR283

You're right, we should restore the default value for this function also :)

**Pull request merged**